static/js/tippyjs.js

```diff
@@ -105,6 +105,11 @@ export function initialize() {
         delay: [300, 20],
     });
 
+    delegate("body", {
+        target: ".message_time",
+        placement: "top",
+    });
+
```

Comment:
We should try to avoid adding `delegate` unless necessary. You can achieve the same behavior here by adding `tippy-zulip-tooltip` class to `.message_time` element. `data-tippy-placement="top"` for placement.

Reply:
Got it, changed. I was confused by this comment:
```
        // Add elements here which are not displayed on
        // initial load but are displayed later through
        // some means.
        ```
and thought it's necessary to add `delegate`.

Reply:
Yeah, we should replace that comment.

Reply:
@amanagr can you help with this? The comment was added in 9c6f5d5946cb605538062028a7feded43931c13d. I'm not sure what it is meant for. Maybe I'm missing something here.

Reply:
It is for special case elements which have different properties than `tippy-zulip-tooltip` class provides. I will change the comment.

Reply:
Opened #19016